src/app/views/examples/table-example/table-example.component.html

```diff
@@ -77,8 +77,8 @@ <h2>Journals</h2>
   <div class="table-container">
     <forge-table
       #journalTable
-      [columnConfigurations]="columnConfigs$ | async"
-      [data]="pagedData$ | async"
+      [columnConfigurations]="(columnConfigs$ | async) ?? []"
```

Comment:
Another case where we may want to consider allowing nulls or migration will require something like this.

Reply:
@DRiFTy17 I'm kind of mixed on `data` and `columnConfigurations` allowing null, since really we can accept a null coming in, but it will never be null coming in.  I feel like maybe this is a case where `?? []` is actually best, since otherwise it will also force null checking unnecessarily on anyone accessing the property if someone has strictNullChecks enabled.